src/content/1.7/modules/concepts/controllers/admin-controllers/tabs.md

```diff
@@ -253,4 +266,94 @@ class example_module_mailtheme extends Module
 
 And now you have your menu link directing to your Symfony controller with a nice url.
 
+### Automatic tab registration
+{{< minver v="1.7.7" title="true" >}}
+
+Modern controllers can also be registered via the `$tabs` property, you don't need to manually create the Tab object in this case, and you can take full advantage of the Symfony routing (no more `_legacy_link`).
+
+Here is an example with a Symfony controller (example comes from the `ps_linklist` module), nothing specific in this controller but you will note the security annotation `@AdminSecurity` that uses `request.get('_legacy_controller')` which will make the link between this controller and the routing configuration.
+
+```php
+// yourmodule/src/Controller/Admin/Improve/Design
+
+namespace PrestaShop\Module\LinkList\Controller\Admin\Improve\Design;
+
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use PrestaShopBundle\Security\Annotation\AdminSecurity;
+use PrestaShopBundle\Security\Annotation\ModuleActivated;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
```

Comment:
```suggestion
// (...)
```

Reply:
I'l just leave `AdminSecurity` as it's useful if you need to copy/paste it